src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResourceTest.kt

```diff
@@ -1481,6 +1481,225 @@ class ReportResourceTest : SqsIntegrationTestBase() {
     }
   }
 
+  @DisplayName("POST /incident-reports/{id}/description-addendums")
+  @Nested
+  inner class AddDescriptionAddendum {
+    private lateinit var url: String
+
+    // language=json
+    private val validPayload = """
+      {
+        "createdAt": "2023-12-05T21:00:00",
+        "createdBy": "USER_1",
+        "firstName": "John",
+        "lastName": "Doe",
+        "text": "Prisoner was released from hospital"
+      }
+    """
+
+    @BeforeEach
+    fun setUp() {
+      url = "/incident-reports/${existingReport.id}/description-addendums"
+    }
+
+    @DisplayName("is secured")
+    @Nested
+    inner class Security {
+      @DisplayName("by role and scope")
+      @TestFactory
+      fun endpointRequiresAuthorisation() = endpointRequiresAuthorisation(
+        webTestClient.post().uri(url).bodyValue(validPayload),
+        "MAINTAIN_INCIDENT_REPORTS",
+        "write",
+      )
+    }
+
+    @DisplayName("validates requests")
+    @Nested
+    inner class Validation {
+      @Test
+      fun `cannot add description addendum to a missing report`() {
+        webTestClient.post().uri("/incident-reports/11111111-2222-3333-4444-555555555555/description-addendums")
+          .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_INCIDENT_REPORTS"), scopes = listOf("write")))
+          .header("Content-Type", "application/json")
+          .bodyValue(validPayload)
+          .exchange()
+          .expectStatus().isNotFound
+
+        assertThatNoDomainEventsWereSent()
+      }
+
+      @Test
+      fun `cannot add description addendum without createdBy field`() {
+        webTestClient.post().uri(url)
+          .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_INCIDENT_REPORTS"), scopes = listOf("write")))
+          .header("Content-Type", "application/json")
+          .bodyValue(
+            // language=json
+            """
+              {
+                "createdAt": "2023-12-05T12:34:56",
+                "firstName": "John",
+                "lastName": "Doe",
+                "text": "Prisoner was released from hospital"
+              }
+            """,
+          )
+          .exchange()
+          .expectStatus().isBadRequest
+
+        assertThatNoDomainEventsWereSent()
+      }
+
+      @Test
+      fun `cannot add description addendum without createdAt field`() {
+        webTestClient.post().uri(url)
+          .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_INCIDENT_REPORTS"), scopes = listOf("write")))
+          .header("Content-Type", "application/json")
+          .bodyValue(
+            // language=json
+            """
+              {
+                "createdBy": "USER_1",
```

Comment:
potentially we don't need to pass the username as we already have it from the token

Reply:
no longer _required_ as of #329